evaluators/evaluator_factory.py

```
@@ -1,28 +1,28 @@
-    """异步评估器工厂类"""
+class EvaluatorFactory:
+    """评估器工厂类"""
     
-    # 可用的异步评估器类型
+    # 可用的评估器类型
     EVALUATOR_TYPES = {
-        "async_academic": AsyncAcademicEvaluator,
-        "async_ragas": AsyncRagasEvaluator
+        "academic": AcademicEvaluator,
+        "ragas": RagasEvaluator
     }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)